packages/components/src/text-control/stories/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import TextControl from '..';
+
+const meta: ComponentMeta< typeof TextControl > = {
+	component: TextControl,
+	title: 'Components/TextControl',
+	argTypes: {
+		onChange: {
+			action: 'onChange',
+			control: { type: null },
+		},
+		value: {
+			control: { type: null },
```

Comment:
@mirka, what's your stance here? At the moment the `value` control is disabled since the story controls `TextControl`'s value via internal state.

Reply:
Sure, I think it's a reasonable pattern to follow for controlled stories.